src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -16,41 +16,41 @@ public enum LanguageVersion
         Default = 0,
 
         /// <summary>
-        /// C# language version 1.0.
+        /// C# language version 1
         /// </summary>
         CSharp1 = 1,
 
         /// <summary>
-        /// C# language version 2.0.
+        /// C# language version 2
         /// </summary>
         CSharp2 = 2,
 
         /// <summary>
-        /// C# language version 3.0.
+        /// C# language version 3
         /// </summary>
         /// <remarks> 
         /// Features: LINQ.
         /// </remarks>
         CSharp3 = 3,
 
         /// <summary>
-        /// C# language version 4.0.
+        /// C# language version 4
         /// </summary>
         /// <remarks> 
         /// Features: dynamic.
         /// </remarks>
         CSharp4 = 4,
 
         /// <summary>
-        /// C# language version 5.0.
+        /// C# language version 5
         /// </summary>
         /// <remarks> 
         /// Features: async.
```

Comment:
It would be nice if a more complete list of features was added for older versions too. Though recent version are the most useful, so that can be done later.